### ISSUE TITLE
Enhancement of declaration of consent to digital transmission of the corona test result

### ISSUE THREAD

**Issue by q-wertz** [corona-warn-app/cwa-wishlist#377]
## Current Implementation
At the moment it is necessary that the medical office crosses the declaration of consent that the test result is transmitted digitally via the app. (see e.g. https://www.kbv.de/html/1150_49115.php).

Many medical offices do not know about it/forget it apparently (happened to me and when looking into the Google Play store comments I think this also happened to many others). Also the App user does not know about it, only sees a warning `Testergebnis liegt noch nicht vor.`.

Also heard from medical staff that they didn't know about it.

## Suggested Enhancement
1. On the App site a pop up after scanning could remind the app user to ask the medical staff if they marked the relevant part
2. On the layout of the consent form it might be good to enhance it in a way that it is clear to the medical staff that they have to make a cross at the relevant position
3. Maybe it is even possible to consent to this via the app after scanning the code? Not sure at which point there is a data leak/privacy issue as I do not really know the exact procedure. It might be possible to scan the QR code, consent to the digital transmission and when the lab scans the QR code it sees whether it should be transmitted digitally or not...

## Expected Benefits
* Faster test results (user does not have to wait for the call of the doctor)
* The user knows it is not the apps fault that he didn't get a respond but a human/organizational one
* User has more control
* Raises acceptance of the app

**Comment by MikeMcC399** [comment]
@q-wertz 

There is some related information to your suggestion on https://www.coronawarn.app/en/faq/#qr_test where it says that you can give feedback about a specific case by sending e-mail to corona-warn-app.opensource@sap.com.

Also, there is another related thread in https://github.com/corona-warn-app/cwa-documentation/issues/400.

**Comment by Ein-Tim** [comment]
Related Issue: https://github.com/corona-warn-app/cwa-documentation/issues/475

**Comment by q-wertz** [comment]
ah sorry searched for existing topics but apparently not good enough

**Comment by Ein-Tim** [comment]
@q-wertz 

I don't think that your issue here is a duplicate of https://github.com/corona-warn-app/cwa-documentation/issues/475. 
Still, in this issue, you can find valuable information why this box is needed, etc. 
I suggest that you read through the issue, especially the explanations from @codejus starting [here](https://github.com/corona-warn-app/cwa-documentation/issues/475#issuecomment-749270167) are very interesting. 
Afterwards you can probably tell whether your suggestion here is feasible or not.
If you should come to the conclusion that your suggestion is not feasible, you can close this issue.
If you come to the conclusion that your suggestion would theoretically be feasible, leave this issue open and one of the friendly community managers/moderators (like @dsarkar) will come back to you here soon.

Have a nice Sunday everyone!

**Comment by q-wertz** [comment]
Ok thanks for the information. From the explanation I think that number 3. is not feasible.
But number 1. (informing the user using a popup after scanning a QR code, that he has to consent on the "Antrag für SARS-COV-2 Testung" to be digitally informed of the test result) might be possible and very simple to implement and might already help a lot.

**Comment by heinezen** [comment]
Also relevant: https://github.com/corona-warn-app/cwa-wishlist/issues/270 (with a very similar discussion) and https://github.com/corona-warn-app/cwa-wishlist/issues/284

----
Corona-Warn-App Open Source Team

**Comment by heinezen** [comment]
@q-wertz 

Thanks for your input. We have mirrored your idea to our internal Jira where it will be discussed by the devs. We'll notify everyone in this PR if we get news on the progress of the issue.

----
Corona-Warn-App Open Source Team
